djedi-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "djedi-react",
-  "version": "5.4.0",
+  "version": "5.4.1",
```

Comment:
I always change the version number in package-lock.json too, to avoid unexpected git diff the next time someone runs `npm install`. It's really weird – npm copies the version number in there when updating the lock file.

Reply:
Ah, I see! Thanks. Will update it as well.